### PR TITLE
Cnft1 3277 UI add patient name column

### PR DESCRIPTION
**Pull request by benlam-ignw**
## Description

AC fulfilled: 
User should be able to see Names as a column, in the patient search results table view.
User should see Names as a column in the ‘Column settings’

## Tickets

https://cdc-nbs.atlassian.net/browse/CNFT1-3277

## Checklist before requesting a review
- [x] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

### CODE REVIEW

**Review by mpeels — APPROVED** [review]

**Review by stevegsa — DISMISSED** [review]

**Review by mpeels — APPROVED** [review]

File: apps/modernization-ui/src/apps/search/patient/result/patientSearchResult.tsx [hunk 1]
```diff
@@ -37,8 +37,29 @@ const displayAddresses = (result: PatientSearchResult): JSX.Element => (
...
+        <div>
+            <b>Legal name</b>
+            <br />
+            {result.legalName && displayName('fullLastFirst')(result.legalName)}
+
+            {result.names
+                .filter((v) => JSON.stringify(v) !== JSON.stringify(result.legalName))
+                .reverse()
+                .map((name, index) => (
+                    <div key={index}>
```
Comment (joeaxcdw) [inline apps/modernization-ui/src/apps/search/patient/result/patientSearchResult.tsx:51]: Use the new <ItemGroup> element in design system here. It already has the right CSS styling per the Figma design.

**Review by joeaxcdw — CHANGES_REQUESTED** [review]
See my comment below

File: apps/modernization-ui/src/apps/search/patient/result/patientSearchResult.tsx (hunk 1 above)
Comment (joeaxcdw) [reply apps/modernization-ui/src/apps/search/patient/result/patientSearchResult.tsx:51]: Also get latest from main and see `displayOtherNames `function. There is some overlap here.

**Review by adamloup-enquizit — CHANGES_REQUESTED** [review]
-I think this story was adding the "Patient name" column as a new column leaving the "Legal name" column in place.-

Designs were either changed or they didn't include the "Legal name".

**Review by joeaxcdw — APPROVED** [review]

**Pull request merged**